airflow/cli/commands/celery_command.py

```diff
@@ -96,6 +99,31 @@ def _serve_logs(skip_serve_logs: bool = False):
         sub_proc.terminate()
 
 
+@after_setup_logger.connect()
+def logger_setup_handler(logger, **kwargs):
+    # Setup levels at which logs go to stderr and stdout if required
+    if conf.has_option("logging", "reset_and_split_logging") and conf.get(
+        "logging", "reset_and_split_logging"
+    ):
+        airflow_formatter = logging.Formatter(conf.get("logging", "log_format"))
+
+        class NoErrorOrAboveFilter(logging.Filter):
+            def filter(self, record):
+                return record.levelno <= logging.WARNING
+
+        below_error_handler = logging.StreamHandler(system_stdout)
+        below_error_handler.addFilter(NoErrorOrAboveFilter())
+        below_error_handler.setFormatter(airflow_formatter)
+
+        from_error_handler = logging.StreamHandler(system_stderr)
+        from_error_handler.setLevel(logging.ERROR)
+        from_error_handler.setFormatter(airflow_formatter)
+
+        logger.handlers.clear()
+        logger.addHandler(below_error_handler)
+        logger.addHandler(from_error_handler)
```

Comment:
Another way would be to do
```
logger.handlers[:] = [below_error_handler, from_error_handler]
```
(as done in _capture_task_logs method)

Since we access `logger.handlers` in `logger.handlers.clear()` anyway, then above can be more concise version of achieving the same.

Reply:
Updated.